sql/mozfun/norm/windows_version_info/udf.sql

```diff
@@ -0,0 +1,82 @@
+CREATE OR REPLACE FUNCTION
+  norm.windows_version_info( os STRING,
+    os_version STRING,
+    windows_build_number INT64 )
+  RETURNS STRING AS (
+    CASE
+      WHEN os NOT IN ('Windows_NT', 'Windows_95', 'Windows_98') THEN NULL
+      WHEN os = 'Windows_95' THEN 'Windows 95'
+      WHEN os = 'Windows_98' THEN 'Windows 98'
+      WHEN os_version = '4.0'
+    AND os = 'Windows_NT' THEN 'Windows NT 4.0'
+      WHEN os_version = '5.0' THEN 'Windows 2000'
+      WHEN os_version = '5.1'
+    OR os_version = '5.2' THEN 'Windows XP'
+      WHEN os_version = '6.0' THEN 'Windows Vista'
+      WHEN os_version = '6.1' THEN 'Windows 7'
+      WHEN os_version = '6.2' THEN 'Windows 8'
+      WHEN os_version = '6.3' THEN 'Windows 8.1'
```

Comment:
Do we want to differentiate these? Or lump them into `Windows 8`?

Reply:
I think for analysis it's fine to have these separated out, if folks find that grouping them becomes something that happens more often then not then perhaps we update it.

In the case of weirdness, it could potentially make it easier to track something down. And I guess MSFT thinks they are different enough to have them separated out.